specs/commandline/blob.md

```diff
@@ -0,0 +1,365 @@
+        │       └── expiry: Sat Jun 23 22:04:01 2035
+        └── signed artifact
+            ├── media type: application/vnd.oci.image.manifest.v1+json
+            ├── digest: sha256:b94d27b9934d3e08a52e52d7da7fac484efe37a5380ee9088f7ace2efcde9
+            └── size: 16724
+```
+
+### Inspect the given detached blob signature with JSON Output
+
+```shell
+notation blob inspect -o json /tmp/my-blob-signature.sig.jws
+```
+
+## Verify detached blob signatures
+The `notation blob verify` command can be used to verify blob signatures. In order to verify signatures, user will need to setup a policy configuration file with Policies scoped to blobs. Below are three examples of how a policy configuration file can be setup for verifying blob signatures.
+
+- The Policy named "blob-verification-policy" is for verifying blob artifacts signed by Wabbit Networks and scoped to `blob-verification-selector`.
+- Policy named "skip-blob-verification-policy" is for skipping verification on blob artifacts scoped to `skip-blob-verification-selector`.
+- Policy "wildcard-blob-verification-policy" is for auditing verification results when user wants to apply a wildcard policy by not providing `--policy-scope` argument in `notation blob verify` command.
+
+```jsonc
+{
+    "version": "1.1",
+    "trustPolicies": [
+        {
+            "name": "blob-verification-policy",
+            "scopes": [ 
+              "blob:blob-verification-selector"
+            ],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": [ 
+              "ca:wabbit-networks",
+            ],
+            "trustedIdentities": [
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Security Tools"
+            ]
+        },
+        {
+            "name": "skip-blob-verification-policy",
+            "scopes": [ "blob:skip-blob-verification-selector" ],
+            "signatureVerification": {
+              "level" : "skip" 
+            }
+        },
+        {
+            "name": "wildcard-blob-verification-policy",
+            "scopes": [ "blob:*" ],
+            "signatureVerification": {
+              "level" : "audit"
+            },
+            "trustStores": ["ca:acme-rockets"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+```
+
+### Verify the detached signature of a blob
+
+Configure trust store and trust policy properly before using `notation blob verify` command.
+
+```shell
+
+# Prerequisites: Signature is produced on the filesystem from `notation blob sign` command.
+# Configure trust store by adding a certificate file into trust store named "wabbit-network" of type "ca"
+notation certificate add --type ca --store wabbit-networks wabbit-networks.crt
+
+# Create a JSON file named "trustpolicy.json" under directory "{NOTATION_CONFIG}".
+
+# Verify the detached signature
+notation blob verify --signature /tmp/my-blob-signature.sig.jws /tmp/my-blob.bin
```

Comment:
Same here, can you add an example of command `notation blob verify --signature /tmp/my-blob-signature.sig /tmp/my-blob.bin`?

Reply:
Same comment as above.